ui/src/app/settings/components/Nav/__snapshots__/Nav.test.js.snap

```diff
@@ -7,6 +7,69 @@ exports[`Nav renders 1`] = `
   <ul
     className="settings-nav__list"
   >
+    <li
+      className="settings-nav__item"
+      key="/configuration"
+    >
+      <strong
+        className=""
```

Comment:
empty `className`

Reply:
I think these are OK, as they will only get a value when the item is active.